modules/configuration/tools/import_projects.php

```diff
@@ -12,32 +21,64 @@
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://www.github.com/aces/Loris/
  */
-
 require_once __DIR__ . "/../../../vendor/autoload.php";
 $client = new NDB_Client();
 $client->makeCommandLine();
 $client->initialize();
+$factory   = NDB_Factory::singleton();
+$config    = $factory->config(__DIR__ . "/../../../project/config.xml");
+$subprojs  = $config->getSettingFromXML("subprojects");
+$db        = $factory->database();
+$optionpos = 1; //The position of the option in the command line.
 
-$factory  = NDB_Factory::singleton();
-$config   = $factory->config(__DIR__ . "/../../../project/config.xml");
-$subprojs = $config->getSettingFromXML("subprojects");
-$db       = $factory->database();
-foreach ($subprojs['subproject'] as $row) {
-    $windowDiff = "optimal";
-    if (isset($row['options']) && isset($row['options']['WindowDifference'])) {
-        $windowDiff = $row['options']['WindowDifference'];
+if (is_null($argv[$optionpos])
+) {
+    echo ("The script needs an argument. The arguments are -s,-p or -a.\r\n");
+    echo ("-s -> Imports only the subprojects from the XML.\r\n");
+    echo ("-p -> Imports only the projects from the XML.\r\n");
+    echo ("-a -> Imports both the subprojects and the projects \r\n");
+    echo ("      from the XML.\r\n");
+    exit (2);
+}
+if ((isset($argv[$optionpos]) && $argv[$optionpos] === "-s")
```

Comment:
@driusan this is the part where subprojects are imported. My comment on the last pull request was:

I am still getting NULL in the WindowDifference column when it is not set in my config.xml, even though you have defined the default in this script as 'optimal'. I think this is the exact same code as we currently have.

My subproject config.xml looks like:

```xml
<subprojects>
   <subproject>
      <id>1</id>
      <title>MCI</title>
      <options>
         <useEDC>false</useEDC>
      </options>
   </subproject>
   <subproject>
      <id>2</id>
      <title>AD</title>
      <options>
         <useEDC>false</useEDC>
      </options>
   </subproject>
</subprojects>
```